cmd/get/get.go

```diff
@@ -45,6 +61,39 @@ func simplePathCommand(path string, description string) *cobra.Command {
 	}
 }
 
+func policyCmd() *cobra.Command {
+	policyCmd := &cobra.Command{
+		Use:   "policies",
+		Short: "Validates existing policies",
+	}
+	simplePathPolicyCommand := func(pathCmd string, args []string, description string) {
+		commands := []string{pathCmd}
+		cobraArgs := cobra.NoArgs
+		if len(args) > 0 {
+			commands = append(commands, args...)
+			cobraArgs = cobra.ExactArgs(len(args))
+		}
+		policyCmd.AddCommand(&cobra.Command{
+			Use:   strings.Join(commands, " "),
+			Short: description,
+			Args:  cobraArgs,
+			RunE: func(cmd *cobra.Command, args []string) error {
+				path := fmt.Sprintf("policy/%s", pathCmd)
+				for i, _ := range args {
+					path = path + "/" + args[i]
+				}
+				return get(path, output)
+			},
+		})
+	}
+	simplePathPolicyCommand("gateway", nil, "Validates if gateway can be created")
+	simplePathPolicyCommand("incominglink", nil, "Validates if incoming links can be creted")
+	simplePathPolicyCommand("outgoinglink", []string{"hostname"}, "Validates if an outgoing link to the given hostname is allowed")
+	simplePathPolicyCommand("expose", []string{"target-type", "target-name"}, "Validates if the given resource can be exposed")
+	simplePathPolicyCommand("service", []string{"name"}, "Validates if service can be created or imported")
+	return policyCmd
```

Comment:
Suggestion for future enhancement would be to have an "inspect" or similar that enumerates the policies applied to the namespace (e.g. or of all the policies defined for the site applied to this namespace). This would also help for namespaces not in the allow list to review what is being applied.

Reply:
Created issue #698 to track this.